sinatra/lesson1/tutorial.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: Introduction to Sinatra
+---
+
+## What is Sinatra?
+
+Sinatra is a super simple framework for making websites using Ruby and HTML.
+It's easy to understand, and a great tool to start learning web development
+with!
+
+Before you continue, make sure you'll completed the [HTML tutorial][html-tut],
+[version control tutorial][vc-tut], and the [Ruby tutorial][ruby-tut].
+
+[html-tut]: http://codebar.github.io/tutorials/html/lesson1/tutorial.html
+[vc-tut]: http://codebar.github.io/tutorials/version-control/introduction/tutorial.html
+[ruby-tut]: http://codebar.github.io/tutorials/ruby/lesson1/tutorial.html
+
+## Installing Ruby
```

Comment:
I think it may best to link to the [setup guide](http://tutorials.codebar.io/general/setup/tutorial.html) rather than get into any detail here about setting up Ruby.